### PR TITLE
[html plugin] new ricos schema

### PR DESCRIPTION
**Pull request by danarish4**

### CODE REVIEW

**Review by shaulgo — APPROVED** [review]
only add url example to the fixtures

**Comment by shaulgo** [comment]
@mharsat Please take a look

File: packages/ricos-content/web/src/converters/draft/toDraft/convertDraftPluginData.ts [hunk 1]
```diff
@@ -208,7 +209,12 @@ const convertButtonData = (data: Partial<ButtonData> & { button }) => {
 };
 
 const convertHTMLData = data => {
-  delete data.config?.size;
+  const { html, url, config = {} } = data;
+  const srcType = html ? 'html' : 'url';
+  data.srcType = srcType;
+  data.src = html ? html : url;
```
Comment (mharsat) [inline packages/ricos-content/web/src/converters/draft/toDraft/convertDraftPluginData.ts:215]: also nice
```suggestion
  data.src = html || url;
```

File: packages/ricos-schema/web/src/main/proto/rich_content/v1/common.proto [hunk 2]
```diff
@@ -65,8 +65,9 @@ message PluginContainerData {
     }
 
     Width width = 1;
-    Alignment alignment = 2;
-    Spoiler spoiler = 3;
+    google.protobuf.Int32Value custom_height = 2;
```
Comment (mharsat) [inline packages/ricos-schema/web/src/main/proto/rich_content/v1/common.proto:68]: This is a breaking change to field indices in common.proto, better add it as index 4

**Review by mharsat — APPROVED** [review]
Amazing! 🚀
Just need a fix in common.proto

File: packages/ricos-schema/web/src/main/proto/rich_content/v1/common.proto (hunk 2 above)
Comment (shaulgo) [reply packages/ricos-schema/web/src/main/proto/rich_content/v1/common.proto]: basa

**Pull request merged**